.github/workflows/language-check.yml

```
@@ -0,0 +1,44 @@
+name: Check README Suffixes
+
+on:
+  pull_request:
+    paths:
+      - 'README-*'
+
+jobs:
+  check-readme-suffixes:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 3.0
+
+    - name: Install dependencies
+      run: |
+        gem install nokogiri
+
+    - name: Check README suffixes
+      run: |
+        # Scrape ISO 639 language codes from the Wiki page
+        wget -qO- https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes | \
+        nokogiri - -e 'puts $_.css(".wikitable td:first-child").map(&:text)' > iso639.txt
```

I tested this locally, and I had to make some changes to make it get just the two letter values. Hope these suggestions make sense.

```suggestion
        wget -qO- https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes | \
        nokogiri -e 'puts $_.css(".wikitable td:nth-child(2)").map(&:text)' | \
        sed -nr '/^.{0,2}$/p' > iso639.txt  # Grab only two letter values
```

---

Ya! that make sense.Thanks for correcting me.